### PR TITLE
Update symfony to 6.4

### LINKED ISSUE #35067: Upgrade to Symfony 6.4
Upgrade from Symfony 6.3 to Symfony 6.4

- https://github.com/symfony/symfony/blob/6.4/UPGRADE-6.4.md

We must prevent the usage of already deprecated methods/code as much as possible.

> [!NOTE]  
> This hunt on deprecated code can be done in separated PRs if the upgrade PR is already too big

### PR DESCRIPTION
**Pull request by M0rgan01**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update Symfony from 6.3 to 6.4
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI / UI tests are green
| UI Tests          | legacy: https://github.com/M0rgan01/ga.tests.ui.pr/actions/runs/7653636573, symfony: https://github.com/M0rgan01/ga.tests.ui.pr/actions/runs/7653637510
| Fixed issue or discussion?     | Fixes #35067
| Related PRs       | -
| Sponsor company   | -

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -55,8 +55,8 @@ framework:
     strict_requirements: ~
   form: ~
   csrf_protection: ~
-  validation: { enable_annotations: true }
-  serializer: { enable_annotations: true }
+  validation: { enable_attributes: true }
+  serializer: { enable_attributes: true }
```
Comment (M0rgan01) [inline app/config/config.yml:59]: It is not possible to have enable_annotations, it is also not possible to have both values.

File: tests/Resources/config/services.yml [hunk 2]
```diff
@@ -47,3 +47,6 @@ services:
 
     security.csrf.token_storage:
       class: Tests\Integration\Utility\CsrfTokenStorage
+
+    Symfony\Component\Serializer\SerializerInterface:
+      alias: serializer
```
Comment (M0rgan01) [inline tests/Resources/config/services.yml:52]: Is now private. The alias is present for a behat test

**Review by jolelievre — APPROVED** [review]
Thanks @M0rgan01

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Congrats @M0rgan01

**Review by boherm — APPROVED** [review]

**Comment by Hlavtox** [comment]
@M0rgan01 knows how to get 💩 done!!! :-)

**Comment by jolelievre** [comment]
Thanks @M0rgan01

**Pull request merged**